src/config.c

```diff
@@ -187,10 +188,25 @@ CONFIG_SETTER(setWorkThreads) {
     QueryError_SetErrorFmt(status, QUERY_ELIMIT, "Number of worker threads cannot exceed %d", MAX_WORKER_THREADS);
     return REDISMODULE_ERR;
   }
+
+  int ret = REDISMODULE_OK;
+  size_t res = workersThreadPool_SetNumWorkers(newNumThreads);
```

Comment:
I don't see any possible ways for this to happen with the current implementation.
This will protect us if someone decides that thpool->n_threads can be changed from the background or any such breakage of the basic assumptions.